source/DomainTheory/BasesAndContinuity/Bases.lagda

```diff
@@ -490,6 +490,10 @@ module _
                        → index-of-compact-basis 𝒷 → ⟨ 𝓓 ⟩
  family-of-basic-opens (_ , β , _) = β
```

Comment:
Could you rename this to `family-of-compact-elements`? The name `family-of-basic-opens` only makes sense when we consider a locale as a dcpo, I think. Moreover, in general, the basic opens for the Scott topology of an algebraic dcpo are given by the upper sets of compact elements, so this name is potentially confusing as we aren't considering that here.

Reply:
> so this name is potentially confusing as we aren't considering that here.

True! Fixed now.